src/wrappers/wrapper.js

```diff
@@ -133,7 +133,7 @@ export default class Wrapper implements BaseWrapper {
 
     const elStyle = window.getComputedStyle(this.element)[style]
     const mockNodeStyle = window.getComputedStyle(mockNode)[style]
-    return elStyle === mockNodeStyle
+    return elStyle && mockNodeStyle && elStyle === mockNodeStyle
```

Comment:
You need to wrap this whole return statement in parentheses and cast them to a Boolean:
```
return !!(elStyle && mockNodeStyle && elStyle === mockNodeStyle)
```

Reply:
ok, i have fixed it